.idea/compiler.xml

```diff
@@ -66,7 +65,7 @@
       <module name="joliec" target="1.8" />
       <module name="jsonrpc" target="1.8" />
       <module name="launchers" target="1.5" />
-      <module name="libjolie" target="11" />
+      <module name="libjolie" target="1.8" />
```

Comment:
Not sure about these changes. I remember at least that we're using APIs in `Optional` that require Java 11. It's very strange that this works... perhaps target is ignored?